### PR TITLE
Fix warp pointer if focus window on another monitor

### PR DESCRIPTION
**Pull request by jtaala**
Fixes a related issue partly described in #837.

Note that in Gnome 45+, PaperWM `alt`+`tab` does indeed switch through all windows across spaces and monitors (by default).  However, an issue does occur that if we switch to (and focus) a window on another monitor via `alt`+`tab` etc., when we move the mouse focus will be pulled to the monitor that the mouse is on.

This PR fixes this by warping the mouse pointer to the new monitor if needed (along with a mouse pointer ripple effect).

This PR also now links the behaviour for showing `all` windows or `only windows from current space` in PaperWM to the Gnome Multitasking setting:

image

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -12,20 +12,30 @@ import * as Config from 'resource:///org/gnome/shell/misc/config.js';
...
 
+/**
+ * Convenience function to get global.  Avoids linter flagging
+ * references to global as no-undef
+ * @returns global
+ */
+export function getGlobal() {
+    // eslint-disable-next-line no-undef
+    return global;
+}
```
Comment (Lythenas) [inline utils.js]: I don't think this is needed just to fix a lint warning. For eslint you should be able to fix the warning by adding `global` to the list of globals configured in `.eslintrc.yml`.

But also I haven't gotten the `.eslint.yml` to work for me. I have only ever worked with js configs for eslint.

**Review by Lythenas — COMMENTED** [review]
Other than the eslint comment, the changes seem fine. But I haven't been able to test them yet.

File: utils.js (hunk 1 above)
Comment (jtaala) [reply utils.js]: Well that's much easier!  I got sick of individually ignoring (with eslint ignore lines) every `global` reference... so thought to do it one place.

Will update the lint config.

Comment (jtaala) [reply utils.js]: For reference, I'm just using https://open-vsx.org/extension/dbaeumer/vscode-eslint within `Code` for development.

I just installed it, did a `npm install -g eslint`, closed/opened `Code` and then added the PaperWM folder to the workspace.

**Pull request merged**